next/src/types/EventProps.d.ts

```diff
@@ -0,0 +1,9 @@
+export type EventProps = {
+  id: number;
+  title: string;
+  audioUrl?: string;
```

Comment:
```suggestion
  audioUrl: string | undefined;
```
上記のようにすると audioUrl のフィールド自体は必要だけど、undefine だよーという定義にしてあげられそうですかね🙏
- `audioUrl?` だと フィールド自体が不要という意味合いになってしまう感じです

Reply:
> audioUrl? だと フィールド自体が不要という意味合いになってしまう感じです

あ、これ僕が完全に勘違いしていました 🙏 
修正します。

というより、audioUrl はデフォルトで空文字になるので、udefined は要らないですね。
普通に `audioUrl` で良さそうなので修正します。

Reply:
色んな差分入っちゃってすみませんが、https://github.com/tramaru/hikido/pull/37/commits/fd993a4122ec1aff1b16d229ea5165172407fa66 で対応しました〜。